src/PKSim.Assets.Images/PKSim.Assets.Images.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="12.0.223" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.223" />
+    <PackageReference Include="OSPSuite.Assets" Version="12.0.0-rTjAe" />
```

Comment:
@Yuri05 Core is updating as I write. I will update again but I wanted to start the PR so that you can review